tests/integration/sync/test_sync_build_in_source.py

```diff
@@ -0,0 +1,72 @@
+import os
+import pytest
+import logging
+import json
+from unittest import skip
+from parameterized import parameterized
+
+from tests.integration.sync.sync_integ_base import SyncIntegBase
+from tests.testing_utils import run_command_with_input
+from samcli.lib.utils.resources import AWS_LAMBDA_FUNCTION
+
+LOG = logging.getLogger(__name__)
+
+
+@skip("Building in source option is not exposed yet. Stop skipping once it is.")
+class TestSyncInfra_BuildInSource_Makefile(SyncIntegBase):
+    dependency_layer = False
+
+    def setUp(self):
+        super().setUp()
+        self.new_files_in_source = [
+            self.test_data_path.joinpath(
+                "code/before/makefile_function_create_new_file/file-created-from-makefile-function"
```

Comment:
Will `/` work in windows?

Reply:
seems like windows tests are passing, and I think I did this because I saw it was done in other places:
https://github.com/aws/aws-sam-cli/blob/develop/tests/integration/sync/test_sync_watch.py#L559

Reply:
done